subprojects/build-cache-http/src/integTest/groovy/org/gradle/caching/http/internal/HttpBuildCacheServiceErrorHandlingIntegrationTest.groovy

```diff
@@ -64,7 +64,7 @@ class HttpBuildCacheServiceErrorHandlingIntegrationTest extends AbstractIntegrat
         withBuildCache().run "customTask"
 
         then:
-        output =~ /Could not store entry .* for task ':customTask' in remote build cache: ${errorPattern}/
+        output =~ /Could not store entry .* in remote build cache: ${errorPattern}/
```

Comment:
I don't think this matters that much. This signals that something has gone (catastrophically) bad with either the network or the disk, or maybe there was an internal error in Gradle. Neither of these cases is helped by knowing which task the problem originated from.

Reply:
I think it matters when the remote cache rejects an artifact because of the size. Is it hard to add the context? I guess currently it is hard since we can't pass anything other than a build cache key into the http cache, right?

Reply:
We can put it in the load command, but I'm still not convinced about this. I think for "too big to cache" events the user should rely on the build scan to debug, not on semi-hidden log messages on the console.

Reply:
I remember us adding the additional context back in the day since we had some problems debugging. We can try without now. We have now better data in build scans, so maybe we don't need it any more.